final_task/rss_reader/database_functions.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import psycopg2
+
+def readingPasswordFromFile() -> str:
+    return "1234"
+
+def checkExistance() -> bool:
+    con = None
+    flag = True
+    try:
+        con = psycopg2.connect(
+            database="postgres",
+            user="postgres",
+            password=readingPasswordFromFile(),
+            host="127.0.0.1",
+            port="5432"
+        )
+        cur = con.cursor()
+        cur.execute("SELECT * from cache")
+        logging.debug("Table exists")
+    except psycopg2.DatabaseError as error:
+        flag = False
+        logging.error(error)
+        logging.debug("Table doesn't exist")
+    finally:
+        if con is not None:
+            con.close()
+        return flag
+
+
+def createTable():
+    con = None
+    try:
+        con = psycopg2.connect(
+            database="postgres",
+            user="postgres",
+            password=readingPasswordFromFile(),
+            host="127.0.0.1",
+            port="5432"
```

Comment:
It is better to have a config file for these parameters, together with password to the db.

Reply:
I changed the way, news are saved, so now I don't need to create config file